backend/corpora/dataset_processing/process.py

```diff
@@ -194,22 +194,27 @@ def create_artifact(
         raise e
 
 
-def create_artifacts(local_filename, dataset_id, artifact_bucket):
+def create_artifacts(local_filename: str, dataset_id: str, artifact_bucket: str, can_convert_to_seurat: bool):
     bucket_prefix = get_bucket_prefix(dataset_id)
-    logger.info("Creating Artifacts.")
+    logger.info(f"Creating artifacts for dataset {dataset_id}...")
```

Comment:
This is a bit redundant since only one dataset will be processed at a time (per cloudwatch group), but I guess it doesn't hurt

Reply:
Ah right, forgot about that.

Reply:
Now I remember why I added this... I thought it might be handy for tracing via Log Insights queries across log streams.

e.g. something like:
```
fields @timestamp, @message
| filter @message like /084311bf-678d-43a2-9890-d6e3c6bae13a/
| sort @timestamp desc
| limit 20
```

to show everything that happened to dataset `084311bf-678d-43a2-9890-d6e3c6bae13a`. Of course, this will require adding the dataset ID to other log statements to be useful.